packages/eslint-plugin-pf-codemods/lib/rules/v5/emptyState-warn-change-structure.js

```diff
@@ -4,20 +4,59 @@ const { getPackageImports, ensureImports } = require("../../helpers");
 module.exports = {
   meta: { fixable: "code" },
   create: function (context) {
-    const package = "@patternfly/react-core";
-    const imports = getPackageImports(context, package);
+    const pkg = "@patternfly/react-core";
+    const imports = getPackageImports(context, pkg);
 
     const includesEmptyState = (arr) =>
       arr.some((specifier) => specifier.imported?.name === "EmptyState");
 
-    const includesIconOrTitle = imports.some((specifier) =>
-      ["EmptyStateIcon", "Title"].includes(specifier.imported.name)
-    );
-
     const titleImport = imports.find(
       (specifier) => specifier.imported.name === "Title"
     );
 
+    const getElements = (baseElement, nameToGet) => {
+      const elementsArray = [];
+
+      const searchForElements = (elementToSearch, elementName) => {
+        const elementObject = elementToSearch.expression ?? elementToSearch;
+        if (elementObject?.openingElement?.name?.name === elementName) {
+          elementsArray.push(elementObject);
+        } else {
+          elementObject?.children?.forEach((child) =>
+            searchForElements(child, elementName)
+          );
+        }
+      };
+
+      searchForElements(baseElement, nameToGet);
+
+      return elementsArray;
+    };
+    const allElements = context
+      .getSourceCode()
+      .ast.body.filter((node) => node.expression?.openingElement);
```

Comment:
here is a problem, because it only gets the nodes in the top level of ast.body

the getElements function looks like it recursively searches for elements inside, maybe some similar function like this is suitable for getting all JSXOpeningElements inside Program